doc/mgr/diskprediction.rst

```diff
@@ -0,0 +1,113 @@
+=====================
+DiskPrediction plugin
+=====================
+
+Disk Prediction plugin is used to collect disk information from Ceph OSD and
+send DiskPrediction server. The disk information includes the following.
+
+- Ceph status
+- I/O operations
+- I/O bandwidth
+- OSD status
+- OSD physical disk smart data
+- Storage utilization
+
+Enabling
+========
+
+Run the following command to enable *diskprediction* module in the Ceph
+environment:
+
+::
+
+    ceph mgr module enable diskprediction
+
+Connection settings
+-------------------
+
+The connection settings can be configured on any machine with the proper cephx
+credentials; they are usually the monitor node with client.admin keyring.
+Run the following command to set up connection between Ceph system and
+DiskPrediction server.
+
+::
+
+    ceph diskprediction config-set <diskprediction_server> <diskprediction_user> <diskprediction_password>
+	
+
+The ``<diskprediction_server>`` parameter is DiskPrediction server name, and it
+could be an IP address if required.
+
+The ``<diskprediction_user>`` and ``<diskprediction_password>`` parameters are the user
+id and password logging in to DiskPrediction server.
+
+
+
+The connection settings can be shown using the following command:
+
+::
+
+    ceph diskprediction config-show
+
+
```

Comment:
I think we need a section here that enumerates exactly what information is being sent to the SaaS API so that users understand what compromise they are making to their security/privacy.

Reply:
@hsiang41 This would be a good place to start: describing what data is passed to the service (and when), and what data comes back.  I didn't realize so much cluster information was being shared!  Users will be very sensitive to this, and clear documentation and transparency is needed.  Thanks!

Reply:
OK, agree. I will refine this document.